README.md

```diff
@@ -33,6 +33,14 @@ npm install -g mbtiles
 tessera mbtiles://./whatever.mbtiles
 ```
 
+_Note_: if your [MBTiles](https://www.mapbox.com/developers/mbtiles/) are in
+"pbf" format (instead of "png"), you may also need:
```

Comment:
This is really "If you want to be able to preview vector tiles (MVT/PBF), you need these modules." (It _should_ serve them up w/o needing those modules, so if it does, that's a problem.)